sw/airborne/modules/sensors/airspeed_sdp3x.c

```diff
@@ -163,13 +198,17 @@ void sdp3x_periodic(void)
   }
 }
 
+#define AUTOSET_NB_MAX 20
```

Comment:
is it really needed to have the auto offset for this sensor ?

Reply:
Most of the time the zero is really zero, however it may be good to have it incase we need it on the filed... I will keep an eye on the utilization during flights, and if we really never use it in real applications, then I will PR again to erase it.